frontend/packages/operator-lifecycle-manager/locales/en/olm.json

```diff
@@ -324,9 +325,9 @@
   "Pending": "Pending",
   "OK": "OK",
   "Failed": "Failed",
-  "This operator provides a custom interface you can include in your console. Make sure you trust this operator before enabling its interface.": "This operator provides a custom interface you can include in your console. Make sure you trust this operator before enabling its interface.",
-  "Enabling console UI extension": "Enabling console UI extension",
-  "This operator will be able to provide a custom interface and run any kubernetes command as the logged in user.  Make sure you trust it before enabling.": "This operator will be able to provide a custom interface and run any kubernetes command as the logged in user.  Make sure you trust it before enabling.",
+  "This operator includes a console plugin which provides a custom interface that can be included in the console. The console plugin will prompt for the console to be refreshed once it has been enabled.  Make sure you trust this console plugin before enabling.": "This operator includes a console plugin which provides a custom interface that can be included in the console. The console plugin will prompt for the console to be refreshed once it has been enabled.  Make sure you trust this console plugin before enabling.",
```

Comment:
```suggestion
  "This operator includes a console plugin which provides a custom interface that can be included in the console. The console plugin will prompt for the console to be refreshed once it has been enabled. Make sure you trust this console plugin before enabling.": "This operator includes a console plugin which provides a custom interface that can be included in the console. The console plugin will prompt for the console to be refreshed once it has been enabled. Make sure you trust this console plugin before enabling.",
```

Reply:
Shouldn't this be?
```suggestion
  "This operator includes a console plugin which provides a custom interface that can be included in the console. The console plugin will prompt the console to be refreshed once it has been enabled.  Make sure you trust this console plugin before enabling.": "This operator includes a console plugin which provides a custom interface that can be included in the console. The console plugin will prompt the console to be refreshed once it has been enabled.  Make sure you trust this console plugin before enabling.",
```

Reply:
^ @itsptk

Reply:
I think "prompt for" is correct here since you're not prompting the console, you're prompting the user to refresh the console.

Reply:
Agreed, thanks for correcting that!